test/external/clone.js

```diff
@@ -60,14 +64,14 @@ if (!argv.disabled) {
 test('test github repos that use `standard`', t => {
   t.plan(pkgs.length)
 
-  fs.mkdirSync(TMP, { recursive: true })
+  mkdirSync(TMP, { recursive: true })
 
   parallelLimit(pkgs.map(pkg => {
     const name = pkg.name
     const url = `${pkg.repo}.git`
-    const folder = path.join(TMP, name)
+    const folder = join(TMP, name)
     return cb => {
-      fs.access(path.join(TMP, name), fs.R_OK | fs.W_OK, err => {
+      access(join(TMP, name), R_OK | W_OK, err => {
```

Comment:
Not sure why this variable wasn't reused before 😄 

```suggestion
      access(folder, R_OK | W_OK, err => {
```